File: Corte1/Python/Ejercicio6/Ejercicio6.py
class Matriz:
    def __init__(self, matriz):
        self.matriz = matriz

    def suma_filas(self):
        for fila in self.matriz:
            print(sum(fila))

    def diagonal_principal(self):
        for i in range(len(self.matriz)):
            print(self.matriz[i][i])


m = Matriz([
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
])

m.suma_filas()
m.diagonal_principal()

File: Corte1/Java/Ejercicio6/Ejercicio6.java
class Matriz {
    private int[][] matriz;

    public Matriz(int[][] matriz) {
        this.matriz = matriz;
    }

    public void sumaFilas() {
        for (int i = 0; i < matriz.length; i++) {
            int suma = 0;
            for (int j = 0; j < matriz[i].length; j++) {
                suma += matriz[i][j];
            }
            System.out.println(suma);
        }
    }

    public void diagonalPrincipal() {
        for (int i = 0; i < matriz.length; i++) {
            System.out.println(matriz[i][i]);
        }
    }
}

public class Main {
    public static void main(String[] args) {
        int[][] datos = {
            {1, 2, 3},
            {4, 5, 6},
            {7, 8, 9}
        };

        Matriz m = new Matriz(datos);

        m.sumaFilas();
        m.diagonalPrincipal();
    }
}
